extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldMixin.example.3.json

```diff
@@ -19,6 +19,7 @@
   "https://ns.adobe.com/experience/journeyOrchestration/previousJumpJourneyID": "c19714bc-4576-4448-94a5-425b7b609911",
   "https://ns.adobe.com/experience/journeyOrchestration/reactionActionID": "c19714bc-4576-4448-94a5-425b7b609911",
   "https://ns.adobe.com/experience/journeyOrchestration/reactionChannel": "email",
-  "https://ns.adobe.com/experience/journeyOrchestration/reactionInteractionType": "open"
+  "https://ns.adobe.com/experience/journeyOrchestration/reactionInteractionType": "open",
```

Comment:
@saurabh111091 Changing the filed names will invalidates existing data set . Please deprecate the existing field by adding "meta:status" : "deprecated" and add a new field instead of changing it

Reply:
@saurabh111091 - Any update on this ? I can still push if you are ok with this breaking change

Reply:
@prabhum2 Thanks for following up. Let me get back to you by Monday. One quick question: Once a field is marked deprecated, can it be removed later on? Otherwise I might request you to merge in the changes in its current state

Reply:
@saurabh111091 - Yes we can remove it as long as no one is using the property. Please let me know if you are going to deprecate or want me to push this as it is .

Reply:
@saurabh111091 - Please let me know when this PR is ready for our review

Reply:
Hi @prabhum2  Yes you can merge these changes in. We do not populate recurrenceIndex currently in our schema in any way. Thanks.